gradle.properties

```diff
@@ -13,3 +13,8 @@ POM_LICENCE_DIST=repo
 
 POM_DEVELOPER_ID=pinterest
 POM_DEVELOPER_NAME=Pinterest, Inc.
+
+# Gradle properties
```

Comment:
maybe also configureondemand?

Reply:
Usually it is discouraged to enable it and most probably will gone in next major release as it will be replaced by instant-execution: https://docs.gradle.org/nightly/userguide/configuration_cache.html